uniffi_bindgen/src/bindings/python/templates/wrapper.py

```diff
@@ -31,6 +33,9 @@
 {% include "EnumTemplate.py" %}
 {%- endfor -%}
 
+# Error definitions
+{% include "ErrorTemplate.py" %}
```

Comment:
@tarikeshaq @rfk The `ErrorTemplate` should generate custom exceptions (based on my Enum on the Rust side) and should have a function that is responsible for:

- Check an error result
- Select the correct custom exception based on the `RustError` code and,
- Raise this exception on the Python side

I didn't quite understand if this is really it 🤔